### PR TITLE
[zh]Add kubelet-credentialprovider.v1beta1.md chinese version

### PR DESCRIPTION
**Pull request by zhangxyjlu**
fixed:#34150
Resync kubelet credentialprovider.v1beta1,add kubelet-credentialprovider.v1beta1.md chinese version

### CODE REVIEW

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
+---
+title: Kubelet 凭据提供程序（CredentialProvider）（v1beta1）
+content_type: tool-reference
+package: credentialprovider.kubelet.k8s.io/v1beta1
+auto_generated: true
```
Comment (Sea-n) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md:5]: Remove this line. Localized files are not auto-generated.

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+   </p>
+<!--
+Each key in the map is a pattern which can optionally contain a port and a path.
+Globs can be used in the domain, but not in the port or the path. Globs are supported
+as subdomains like '<em>.k8s.io' or 'k8s.</em>.io', and top-level-domains such as 'k8s.<em>'.
+Matching partial subdomains like 'app</em>.k8s.io' is also supported. Each glob can only match
+a single subdomain segment, so *.io does not match *.k8s.io.
+-->
+   <p>
+   映射中每个键值都是一个正则表达式，可以选择包含端口和路径。，可以选择包含端口和路径。
```
Comment (Sea-n) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md:136]:
```suggestion
   映射中每个键值都是一个正则表达式，可以选择包含端口和路径。
```

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+<code>string</code>
+</td>
+<td>
+<!--
+   image is the container image that is being pulled as part of the
+credential provider plugin request. Plugins may optionally parse the image
+to extract any information required to fetch credentials.
+-->
+   <p>
+   <code>image</code> 是容器映像，作为凭据提供程序插件请求的一部分。
```
Comment (Sea-n) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md:52]:
```suggestion
   <code>image</code> 是容器鏡像，作为凭据提供程序插件请求的一部分。
```

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+title: Kubelet CredentialProvider (v1beta1)
+content_type: tool-reference
+package: credentialprovider.kubelet.k8s.io/v1beta1
+auto_generated: true
+-->
+
+<!--
+## Resource Types
+-->
+## 资源类型
```
Comment (Sea-n) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md:17]: Add link anchor.
```suggestion
## 资源类型   {#resource-types}
```

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+## `CredentialProviderRequest`     {#credentialprovider-kubelet-k8s-io-v1beta1-CredentialProviderRequest}
+
+<!--
+CredentialProviderRequest includes the image that the kubelet requires authentication for.
+Kubelet will pass this request object to the plugin via stdin. In general, plugins should
+prefer responding with the same apiVersion they were sent.
+-->
+<p>
+CredentialProviderRequest 包括 kubelet 需要进行身份验证的镜像。
+Kubelet 会通过标准输入将此请求对象传递给插件。一般来说，插件倾向于用它们发送的相同的 apiVersion 来响应。
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md:30]:
```suggestion
Kubelet 会通过标准输入将此请求对象传递给插件。一般来说，插件倾向于用它们所收到的相同的 apiVersion 来响应。
```

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+<a href="#credentialprovider-kubelet-k8s-io-v1beta1-PluginCacheKeyType"><code>PluginCacheKeyType</code></a>
+</td>
+<td>
+<!--
+cacheKeyType indiciates the type of caching key to use based on the image provided
+in the request. There are three valid values for the cache key type: Image, Registry, and
+Global. If an invalid value is specified, the response will NOT be used by the kubelet.
+-->
+   <p>
+   <code>cacheKeyType</code> 是请求中镜像提供的要使用的缓存键的类型。缓存键类型有三个有效值：
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md:90]:
```suggestion
   <code>cacheKeyType</code> 表明基于请求中所给镜像而要使用的缓存键类型。缓存键类型有三个有效值：
```

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+<td>
+<!--
+cacheDuration indicates the duration the provided credentials should be cached for.
+The kubelet will use this field to set the in-memory cache duration for credentials
+in the AuthConfig. If null, the kubelet will use defaultCacheDuration provided in
+CredentialProviderConfig. If set to 0, the kubelet will not cache the provided AuthConfig.
+-->
+   <p>
+   <code>cacheDuration</code> 表示所提供的凭据应该被缓存的时间。kubelet 使用这个字段为
+   AuthConfig 中的凭据设置内存中的缓存时间。如果为空，kubelet 将使用 CredentialProviderConfig
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md:107]: 英文中这个地方处理的不好。

```suggestion
   `auth` 中的凭据内存中数据的缓存时间。如果为空，kubelet 将使用 CredentialProviderConfig
```

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+   中提供的 defaultCacheDuration。如果设置为 0，kubelet 将不对提供的 AuthConfig 进行缓存。
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md:108]:
```suggestion
   中提供的 defaultCacheDuration。如果设置为 0，kubelet 将不会缓存所提供的 `auth` 数据。
```

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+</td>
+<td>
+<!--
+   auth is a map containing authentication information passed into the kubelet.
+Each key is a match image string (more on this below). The corresponding authConfig value
+should be valid for all images that match against this key. A plugin should set
+this field to null if no valid credentials can be returned for the requested image.
+-->
+   <p>
+   <code>auth</code> 是一个包含传递到 kubelet 的身份验证信息的映射。
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md:123]:
```suggestion
   <code>auth</code> 是一个映射，其中包含传递到 kubelet 的身份验证信息。
```

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+<!--
+Each key in the map is a pattern which can optionally contain a port and a path.
+Globs can be used in the domain, but not in the port or the path. Globs are supported
+as subdomains like '<em>.k8s.io' or 'k8s.</em>.io', and top-level-domains such as 'k8s.<em>'.
+Matching partial subdomains like 'app</em>.k8s.io' is also supported. Each glob can only match
+a single subdomain segment, so *.io does not match *.k8s.io.
+-->
+   <p>
+   映射中每个键值都是一个正则表达式，可以选择包含端口和路径。，可以选择包含端口和路径。
+   通配符可以在域中使用，但不能在端口或路径中使用。
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md:136]:
```suggestion
   域名部分可以包含通配符，但在端口或路径中不能使用通配符。
```

**Comment by zhangxyjlu** [comment]
I have finished modifying your review comments，Thank you very much for your guidance@[Sea-n](https://github.com/Sea-n)

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md [hunk 11]
```diff
@@ -0,0 +1,253 @@
...
+<tr><td><code>username</code> <B><!--[Required]-->[必需]</B><br/>
+<code>string</code>
+</td>
+<td>
+<!--
+   <p>username is the username used for authenticating to the container registry
+An empty username is valid.</p>
+-->
+   <p>
+   <code>username</code> 是用于对容器注册表进行身份验证的用户名。空的用户名是有效的。
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md:222]:
```suggestion
   <code>username</code> 是用于向容器仓库进行身份验证的用户名。空的用户名是合法的。
```

**Comment by zhangxyjlu** [comment]
I have finished modifying your review comments，Thank you very much for your guidance@[tengqm](https://github.com/tengqm)

**Comment by mengjiao-liu** [comment]
/approve
Thanks!

**Pull request merged**